### ISSUE TITLE
Auto analysis ruins Go binary function names

### ISSUE THREAD

**Issue by padorka** [NationalSecurityAgency/ghidra#2248]
**Describe the bug**
After analysis of a non-stripped Linux Go binary, the function names are shortened and incorrect. 

**To Reproduce**
1. Create a simple hello world in Go:

```
package main

import "fmt"

func main() {
	fmt.Printf("hello, world\n")
}
```

2. compile it on Linux (or specify the arch):
go build hello.go -o hello 

3. load to Ghidra and analyse with default settings

**Screenshots**
reflect.(*funcType).NumOut changed to reflect.
![func_double](https://user-images.githubusercontent.com/66425539/92249871-03569880-eecb-11ea-8006-9cb075b7e103.png)

**Attachment**

[hello_go.zip](https://github.com/NationalSecurityAgency/ghidra/files/5175365/hello_go.zip)

**Environment:**
 - OS: Ubuntu 20.0.4.1 LTS
 - Java Version: 11.0
 - Ghidra Version: 9.1.2
- Go: 1.15

**Comment by ghidra1** [comment]
It appears that may be resolved for Ghidra 9.2 as it currently exists in the master branch.

**Comment by padorka** [comment]
Cool. Thank you.
